redwood-lazylayout-view/src/main/kotlin/app/cash/redwood/lazylayout/view/ViewLazyList.kt

```diff
@@ -253,16 +245,20 @@ internal open class ViewLazyList(context: Context) : LazyList<View> {
     }
   }
 
-  private class Container(context: Context) : FrameLayout(context) {
+  class Container(context: Context) : FrameLayout(context) {
+    init {
+      layoutParams = LayoutParams(MATCH_PARENT, WRAP_CONTENT)
```

Comment:
Do these values need to flip based on orientation?

Reply:
Yep, they do. Doing that in a follow-up!